### DISCUSSION TITLE
Cannot remove custom domain

### DISCUSSION THREAD

**Discussion by temery1213** [community/community#68553] [Other Feature Feedback, Questions, & Ideas]
Pages

Question

I'm a bit new to this and was trying to publish my site with a custom domain. I was just messing around and ended up redirecting my page to another site. I deleted the domain name in the pages section of settings and already checked for a cname. Logs say its been deleted and I do not see it. I also tried clearing my cache and accessing my page at temery1213.github.io from multiple browsers and it still redirects?

**Comment by Primus66** [comment]
Incognito/Private Browsing:
	•	Try accessing your GitHub Pages site using a private or incognito browsing window. This can help ensure that your browser isn’t using cached information.
